lib/bloc/notification/ct_notification_bloc.dart

```diff
@@ -29,21 +33,22 @@ class CTNotificationsBloc
   }
 
   Stream<NotificationsBlocState> _fetchNotificationsInternal(
-      {int pageNo = 1}) async* {
+      {int pageNo = 0}) async* {
     yield state.rebuild((b) => b
       ..loading = true
       ..pageNo = pageNo);
 
     try {
-      var notifications = await ApiRepository.getNotificationsList(pageNo);
+      var notifications =
+          await _fetchNotificationsUseCase.execute(pageIndex: pageNo);
```

Comment:
@matthiasbruns page no can't be 0

Reply:
the use case handles the offset

Reply:
where are the page no's getting incremented ?

Reply:
line 29

```kotlin
      case NotificationsBloEvent.fetchMore:
        yield* _fetchNotificationsInternal(pageNo: state.pageNo + 1);
        break;

```